src/aeso_fcode_to_fate.erl

```diff
@@ -179,63 +183,79 @@ lit_to_fate(L) ->
         {contract_pubkey, K} -> aeb_fate_data:make_contract(K);
         {oracle_pubkey, K}   -> aeb_fate_data:make_oracle(K);
         {oracle_query_id, H} -> aeb_fate_data:make_oracle_query(H);
+        {contract_code, C}   ->
+            FCode = maps:get(C, Env#env.child_contracts),
+            SCode = compile(Env#env.child_contracts, FCode, Env#env.options),
+            ByteCode = aeb_fate_code:serialize(SCode, []),
+            {ok, Version} = aeso_compiler:version(),
+            Code = #{byte_code => ByteCode,
+                     compiler_version => Version,
+                     contract_source => "child_contract_src_placeholder",
+                     type_info => [],
+                     fate_code => SCode,
+                     abi_version => aeb_fate_abi:abi_version(),
+                     payable => maps:get(payable, FCode)
+                   },
+            Serialized = aeser_contract_code:serialize(Code),
+            aeb_fate_data:make_contract_bytearray(Serialized);
```

Comment:
Future optimizations:
- don't compile child contract for every literal
- don't duplicate the literal for every `Chain.create`

Reply:
- sneak in `clone` if the contract was created previously